winappdbg/win32/kernel32.py

```diff
@@ -3707,8 +3707,8 @@ def ReadProcessMemory(hProcess, lpBaseAddress, nSize):
     lpBuffer            = ctypes.create_string_buffer(b'', nSize)
     lpNumberOfBytesRead = SIZE_T(0)
     success = _ReadProcessMemory(hProcess, lpBaseAddress, lpBuffer, nSize, byref(lpNumberOfBytesRead))
-    if not success and GetLastError() != ERROR_PARTIAL_COPY:
-        raise ctypes.WinError()
+    #if not success and GetLastError() != ERROR_PARTIAL_COPY:

```

Comment:
This was kinda important to keep the memory dumps from failing on corrupted processes, why was it removed?